scout/constants/indexes.py

```diff
@@ -29,15 +29,6 @@
         ),
     ],
     "variant": [
-        IndexModel(
-            [
-                ("case_id", ASCENDING),
-                ("category", ASCENDING),
-                ("rank_score", DESCENDING),
```

Comment:
It is sure not used often! Doesn't the `delete_variants_query` function kind of use it though? Is it fast enough anyway? Or perhaps it could be changed to use the `variant_type` as well - so that that index is used instead.

Also I can't figure what uses the other non-variant_type partner `caseid_variantrank` is being used by, but never mind. 😅

Reply:
Looks like nothing is using it, bah

Reply:
Based on reading the `delete_variants_query()` or based on low usage reported by `mongod`?

Reply:
the caseid_variantrank I meant, sorry, not the whole index

Reply:
The [delete_variants_query](https://github.com/Clinical-Genomics/scout/blob/a4aac05a30d9b262739305c16dca6ab4ada77332/scout/adapter/mongo/query.py#L63) is using indexes with 
- case_id
- _id
- category
- rank score


We could add variant_type if it's slower. I don't know, we never use that command basically..

Reply:
We have reasoned that last time we used the delete variants command we have used it without the rank score param and it worked fine, so it should be fine to remove the index!